README.md

```diff
@@ -168,6 +168,30 @@ ruby ./input.rb
 # paste in JIRA keys, one line at a time
 ```
 
-#### Potential bundle issues
+#### Build a Git Branch name from a Jira Issue
+The file branch_name.rb can be used to create a git branch name from a Jira issue.
 
-As of `2018-07-09` (YYYY-MM-DD), `bundle install` fails due to `scpt-1.0.2` on ruby version `ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-darwin17]`. This issue should be resolved in the future once a new gem version is released, but meanwhile the issue can be fixed manually by telling the Gemfile to pull from the `develop` branch of `https://github.com/BrendanThompson/rb-scpt`. See `https://github.com/nburwell/jira-glue/tree/fix_rb-scpt-1.0.2_breaks` for an updated Gemfile that includes this change. [Note: Since this fix should be temporary, the Gemfile in master was left as it was]
+The name is created using the current year and month, followed by the jira issue key followed by the 
+summary/description with most non-alphanumeric characters and spaces turned to underscores.
+
+##### Example Branch Name:
+
+Given the following:
+* Current Date: July 4th 2018 (07/04/18)
+* Jira Issue Number: STORY-123
+* Jira Issue Description: Fix Jira-glue bug
+
+The branch name would be the following:
+```
+1807/STORY-123_fix_jira_glue_bug
+```
+
+##### Sample Output:
+```
+ruby ./branch_name.rb STORY-123
+YYMM/STORY-123_jira_issue_description
```

Comment:
this date format is very Invoca specific (and having the prefix at all).  Preference would be to allow the format to be overridden in the config (and if left blank in config, omit it entirely).  If the key is not in the config at all, I'm fine (I think) with the default behavior being YYMM (to allow for easy opt in by existing developers)